### DISCUSSION TITLE
How to undo a bootstrap

### DISCUSSION THREAD

**Discussion by jimsnab** [fluxcd/flux2#1271] [Q&A]
While getting to know flux2 and trying to find my way, I am trying to adapt the Getting Started instructions to what I want to accomplish. I had run the bootstrap command once without --path, and now, when I run the following derived from Getting Started

```
flux bootstrap git --branch main --private-key-file /home/flux/.ssh/id_rsa --url ssh://git@kube-master.cluster.dev:/git/hello.git --path=./clusters/my-cluster
```

I am getting the following error:

```
✗ sync path configuration ("./clusters/my-cluster") would overwrite path ("./") of existing Kustomization
```
I've tried this

```
flux get kustomizations all
✗ no Kustomization objects found in flux-system namespace
```

How do I clear out my initial bootstrap?

**Comment by stefanprodan** [comment]
Use `flux uninstall` to start fresh.

**Reply by jimsnab** [reply]
Is that the only way?
